data/sql/updates/pending_db_world/rev_1626260907562417400.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1626260907562417400');
+UPDATE gossip_menu SET MenuID = 7434 WHERE MenuID = 7983 AND TextID = 9039;
```

Comment:
```suggestion
UPDATE `gossip_menu` SET `MenuID` = 7434 WHERE `MenuID` = 7983 AND `TextID` = 9039;
```
https://www.azerothcore.org/wiki/sql-standards#general-standards